### ISSUE TITLE
Changing language (french)

### ISSUE THREAD

**Issue by xelfe** [explosion/spaCy#919]
I saw that french is now supported and I need to use it but I dont find how. see below:

command;

>  pip install spacy
>  python -m spacy download fr

return;

>  Compatibility error
>   No compatible model found for fr (spaCy v1.7.2).

## My Environment

* **Python version:** 2.7.12
* **Platform:** Linux-4.4.0-66-generic-x86_64-with-elementary-0.4-loki
* **spaCy version:** 1.7.2
* **Installed models:**

**Comment by oroszgy** [comment]
AFAIK there is no supported model yet, so just go ahead and 

```
from spacy.fr import French
nlp = French()
```

**Comment by ines** [comment]
Yes, that's correct – while spaCy supports French tokenization, there's not actually a French model yet. But you can still use the above code to use spaCy in French (or any [other supported language](https://spacy.io/docs/api/language-models#alpha-support)).

**Comment by younes200** [comment]
Hi, with 
```
from spacy.fr import French
nlp = French()
```
It end up with the error `IOError: [Errno 2] No such file or directory: 'rasa_nlu/lib/python2.7/site-packages/spacy/data/fr/vocab/strings.json'`
Any idea ? thank you.

**Comment by oroszgy** [comment]
Smells like a version mismatch. What spacy version are you running?

**Comment by younes200** [comment]
it's spacy==1.7.2

**Comment by ines** [comment]
@younes200 This issue should be fixed in the latest version 1.8.0. (I think this was actually a bug that caused spaCy to try to load a model, even when importing the `Language` class directly.)

**Comment by younes200** [comment]
thanks @ines
